jumpscale/sals/zos/kubernetes.py

```diff
@@ -23,6 +23,7 @@ def add_master(
         size: int,
         ssh_keys: List[str],
         pool_id: int,
+        public_ip: int
```

Comment:
public_ip here is the workload id for the public ip reservation correct? if so please rename it to public_ip_wid or something more clear

Reply:
I can do this but this will require changes in zos and the explorer.